content/class/Animated.md

```diff
@@ -0,0 +1,80 @@
+**Project NewCaw** \
+Author: Frederick Schenk
+
+# Content/Animated
+
+![API: Twitter 1](https://img.shields.io/badge/API-Twitter%201-lightgrey?style=flat-square) ![API: Twitter 2](https://img.shields.io/badge/API-Twitter%202-blue?style=flat-square) ![API: Mastodon](https://img.shields.io/badge/API-Mastodon-purple?style=flat-square)
+
+```c#
+public interface Animated : Media
+```
+
+`Animated` is a subclass of [`Media`](Media.md) which manages a animated picture (gif) from a platform.
+
+> From a technically standpoint `Animated` is just a video without audio, but since it is intended to loop and has different controls it is a separate subclass.
```

Comment:
I also want to make `Animated` to return a video file.

This is again a thing coming from "place all important information in the backend and let the frontend just display it", so it would make sense to have a class for "Animated GIF's", despite it is basically the same as `Video`.